test/Microsoft.Data.Sqlite.Tests/SqliteConnectionTest.cs

```diff
@@ -538,7 +538,7 @@ public void CreateFunction_works_when_parameter_null_but_type_long()
                 var ex = Assert.Throws<SqliteException>(() => connection.ExecuteScalar<long>("SELECT test(NULL);"));
 
                 Assert.Equal(
-                    Resources.SqliteNativeError(raw.SQLITE_ERROR, new InvalidCastException().Message),
+                    Resources.SqliteNativeError(raw.SQLITE_ERROR, Resources.CalledOnNullValue),
```

Comment:
Hmm, the error message doesn't make much sense in this context. Can we customize it for SqliteParameterBinder?

Reply:
Okay, done in the newest revision.